apiml-security-common/src/main/java/org/zowe/apiml/security/common/verify/CertificateValidator.java

```diff
@@ -63,8 +58,7 @@ public boolean isTrusted(X509Certificate[] certs) {
         List<Certificate> trustedCerts = trustedCertificatesProvider.getTrustedCerts(proxyCertificatesEndpoint);
```

Comment:
should we include here the shortcut to not validate if the url is not defined from the v3?

Reply:
good point, we can